### PR TITLE
Tickets/instrm 1195

### PR DESCRIPTION
**Pull request by yukimoritani**

### CODE REVIEW

File: data/fiberids/grandfibermap.20210314.txt [hunk 1]
```diff
@@ -0,0 +1,2599 @@
+\cob  fld   cf mod  mf   cm cbrd  brd mtp      x        y       r   sp  fh  sfib  fiberid  sunss   id(A)        id(C)        id(BA)       id(BC)
```
Comment (PaulPrice) [inline data/fiberids/grandfibermap.20210314.txt:1]: Probably not terribly important, but `fiberid` should be spelled `fiberId`.

File: data/fiberids/grandfibermap.20210314.txt [hunk 2]
```diff
@@ -0,0 +1,2599 @@
...
+ nan  nan  nan nan nan  nan nan nan  nan      nan      nan     nan  1   1   eng       1    emp  EN-1-1-1-x   nan          nan          nan
```
Comment (PaulPrice) [inline data/fiberids/grandfibermap.20210314.txt:2]: `nan` values look strange. Do they read cleanly? Can we write something else? A dash, or ellipsis, perhaps?

File: data/fiberids/README.txt [hunk 3]
```diff
@@ -2,15 +2,19 @@ The columns in the grandfibermap are:
...
+    cbrd : Cobra-in-board (1..29). Each board has either 29 or 28 cobras.
+    brd : Cobra board id (1..84). One Cobra module has two boards.
+    mtp: Cobra module id associated with MTP ferrule. There are 84 of these, numbered 1 through 42 with A and B suffixes.
     x,y: Position of the center of the cobra on the focal plane.
     r: Radius of the center of the cobra.
     sp: Spectrograph that the cobra feeds (1..4)
     fh: Fiber hole (1..651). This is the position in the spectrograph slit head.
-    sfib: Science fiber (1..2394). This is a unique identifier for each science fiber.
-    id: Identifier of the USCONNEC connector hole at the Cable B-C interface.
+    sfib: Science fiber (1..2394). This is a unique identifier for each science fiber. "eng" is engineering fibers, and "emp" is empty hole (i.e. blank fibers).
```
Comment (PaulPrice) [inline data/fiberids/README.txt:15]: Do we need to list empty holes? I think we only care about holes with fibers in them.

**Review by PaulPrice — COMMENTED** [review]
I think you need to upgrade the code in `fiberids.py` to match the new format:
```
>>> from pfs.utils.fiberids import FiberIds
>>> fdb = FiberIds()
>>> fdb.fiberId
array([  1,   2,   3, ..., 649, 650, 651], dtype=uint16)
>>> fdb.fiberId.max()
651
```

File: data/fiberids/README.txt (hunk 3 above)
Comment (yukimoritani) [reply data/fiberids/README.txt:15]: Well, how do you distinguish a given slit hole is originally blank, or filled but not unilluminated? I'm not sure if there is  no use-case at all where blank holes are referred. And In INSTRM-1138, having all slit holes is suggested.

File: data/fiberids/grandfibermap.20210314.txt (hunk 2 above)
Comment (yukimoritani) [reply data/fiberids/grandfibermap.20210314.txt:2]: Thank you for suggestion. I'll use dash.

**Pull request merged**